submissions/LuckyDnepr/Hooli-popup-task/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles-menu.css">
+    <title>Hooli menu</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <div class="logo">LOGO</div>
+            <nav class="navigation">
+                <a href="#User" id="nav_item_user" class="nav_item_icon"></a>
+                <a href="#Notification" id="nav_item_notify" class="nav_item_icon"></a>
+                <input id="nav_popup_check" type="checkbox">
+                <label for="nav_popup_check" id="nav_popup_label">
+                    <img class="nav_icons" src="./icons/popup-button.png" alt="popup-menu">
+                </label>
+                <a href="#Mail" class="nav_item">Mail</a>
```

Comment:
When the user opens the popup and presses "TAB", the focus is not on the first link (_Application_). It's is not expected. 

Expected navigation using keyboard:
1. User open popup and click `TAB` - **focus on the first link in popup**
2. User not open popup and click `TAB`  - **focus on the bell**

You can fix it by changing tag order in html. (Example: move bell and user `a` tags under the popup).